### PR TITLE
Add workflow to enable CI on the package

### PR DESCRIPTION
**Pull request by rajasvu**
1. Added a workflow configuration to enable continuous integration for code commits and pull-request changes on main branch.
2. Tentative scope of integration is to run the tests using pytest for python 3.8 version

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# These are supported funding model platforms
-
-open_collective: pylabel
```
Comment (alexheat) [inline .github/FUNDING.yml:3]: We should keep this file. It should not cause a problem.

File: .github/workflows/push-pr-ci.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest
```
Comment (alexheat) [inline .github/workflows/push-pr-ci.yml:23]: pytest is part of the requirements so we don't need to install it separately

**Review by alexheat — CHANGES_REQUESTED** [review]
I made 2 comments. Please address and then I will merge it

File: .github/workflows/push-pr-ci.yml (hunk 2 above)
Comment (rajasvu) [reply .github/workflows/push-pr-ci.yml]: Fixed the comment as part of [commit](https://github.com/pylabel-project/pylabel/pull/8/commits/5db3cd09bda02686306e2dcccbd6e00f9d75892b)

File: .github/FUNDING.yml (hunk 1 above)
Comment (rajasvu) [reply .github/FUNDING.yml:3]: Fixed as suggested in the [commit](https://github.com/pylabel-project/pylabel/pull/8/commits/9470c31e081b77fb3c07cd752b91900b14cb017f)

**Review by alexheat — APPROVED** [review]
Thank you

**Comment by rajasvu** [comment]
Fixed all the comments as recommended. Please review and do the needful approval of the pull-request

**Pull request merged**